content/rs/references/cli-utilities/rladmin/node/addr.md

```diff
@@ -0,0 +1,50 @@
+---
+Title: rladmin node addr set
+linkTitle: addr
+description: Sets the internal IP address of a node.
+weight: $weight
+alwaysopen: false
+toc: "true"
+headerRange: "[1-2]"
+categories: ["RS"]
+aliases:
+---
+
+`rladmin node addr set` sets the internal IP address of a node. The internal IP address can only be set with `rladmin node addr` when the node is down.
+
+```sh
+rladmin node <id>
+        addr set <ip_addr>
+```
+
+### Parameters
+
+| Parameter | Type/Value                     | Description                                                                                   |
+|-----------|--------------------------------|-----------------------------------------------------------------------------------------------|
+| node      | node ID                        | Sets internal IP address of the specified node                                                |
+| ip_addr   | IP address                     | Sets specified IP address as the internal IP of the node                                      |
+
+### Returns
+
+Returns `Updated successfully` if the IP address was set.
+
+Use [`rladmin status nodes`]({{<relref "/rs/references/cli-utilities/rladmin/status#status-nodes">}}) to verify the internal IP address was changed.
```

Comment:
Consider starting with action first, i.e. "Verify that the internal IP address has changed, ..."